fusion-plugin-i18n/src/node.js

```diff
@@ -100,9 +134,21 @@ const pluginFactory: () => PluginType = () =>
           ctx.template.htmlAttrs['lang'] = localeCode;
         } else if (ctx.path === '/_translations') {
           const i18n = plugin.from(ctx);
-          const keys = querystring.parse(ctx.querystring).keys || '';
-          const translations = keys.split(',').reduce((acc, key) => {
-            acc[key] = i18n.translations && i18n.translations[key];
+          const keys = JSON.parse(
+            querystring.parse(ctx.querystring).keys || '[]'
```

Comment:
Changed this to a stringified array to pass both strings and arrays to translations endpoint. No longer can break on `,`.